tests/src/psa_exercise_key.c

```diff
@@ -822,7 +822,11 @@ int mbedtls_test_psa_exported_key_sanity_check(
     }
     else
 #endif /* MBEDTLS_ECP_C */
-
+    if( PSA_KEY_TYPE_IS_DH_PUBLIC_KEY( type ) || PSA_KEY_TYPE_IS_DH_KEY_PAIR( type ) )
+    {
+        TEST_LE_U( exported_length, PSA_BITS_TO_BYTES( bits ) );
```

Comment:
Why less than or equal to bits?
Shouldn't we have the same the same two conditions as above, so
```
        TEST_ASSERT( exported_length <=
                     PSA_EXPORT_PUBLIC_KEY_OUTPUT_SIZE( type, bits ) );
        TEST_ASSERT( exported_length <=
                     PSA_EXPORT_PUBLIC_KEY_MAX_SIZE );
```
?

Reply:
Yes, this should be valid now also for FFDH.